projects/etos_suite_runner/src/etos_suite_runner/lib/otel_tracing.py

```diff
@@ -46,7 +46,7 @@ class EnvVarContextGetter(Getter):
     def get(self, carrier, key):
         """Get value using the given carrier variable and key."""
         value = os.getenv(carrier)
-        if value is not None:
+        if value is not None and value != "":
```

Comment:
"if value:"  

or

value not in [None, ""]

?

Reply:
1. Explicit is better
2. I believe that readability is higher in my version